Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1021,6 +1128,13 @@ void function OnServerSelected( var button )
 		}
 	}
 
+	// Make Northstar aware news mod have been added
+	if (downloadedMods > 0)
+	{
+		print("Some new mods have been downloaded, reloading mods.")
+		NSReloadMods();
```

Comment:
```suggestion
		NSReloadMods()
```
`NSReloadMods()` is really kinda bad for actually reloading mods properly, this needs to be fixed in native though, just saying this as a warning

Reply:
Do you know a better solution to reload mods?

Reply:
`NSReloadMods()` *should* be the best way, but its a bit shit atm, needs to be improved to support reloading things like rpaks, models, etc.

Reply:
Ok, so it's no problem if I invoke this here :)